app/presenters/portfolio_case_presenter.rb

```diff
@@ -14,7 +25,8 @@ def taxpayer_name?
 
   private
 
-  def tribunal_case
-    __getobj__
+  def case_remaining_days
+    expiration_date = created_at.days_since(Rails.configuration.x.cases.expire_in_days)
+    (DateTime.now...expiration_date).count
```

Comment:
See my comment about `<= 0` above. Another option would be to [x, 0].max this, before we return it, so that remaining days can never be negative